### PR TITLE
[11.0][MIG] base_maintenance_group (split in 2 separate modules)

### PR DESCRIPTION
**Pull request by astirpe**
While porting `base_maintenance_config` to V11,  I'm splitting it in two different modules:

- `base_maintenance_config`, containing only the general configuration view;
- `base_maintenance_group`, containing all the stuff related the newly added groups.

### CODE REVIEW

File: base_maintenance_group/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+This module provides a security group category 'Maintenance' with two new user
+groups:
+
+- 'User'
+- 'Manager'.
```
Comment (dalonsod) [inline base_maintenance_group/readme/DESCRIPTION.rst]: Should this be "Full Access"?

**Review by dalonsod — CHANGES_REQUESTED** [review]

File: base_maintenance_group/readme/DESCRIPTION.rst (hunk 1 above)
Comment (astirpe) [reply base_maintenance_group/readme/DESCRIPTION.rst]: Right, thanks!

**Review by dalonsod — APPROVED** [review]
Functionally & code review 👍

**Comment by dalonsod** [comment]
@astirpe when merged, would you migrate it to v12? If not, I'd interested to

**Comment by astirpe** [comment]
@dalonsod  Yes, my intention was to migrate it to V12 soon.
Anyway if you like to open a PR for V12 just go ahead (preserving the commit history) and I will review it! :+1:

**Comment by dalonsod** [comment]
@astirpe as soon as this PR is merged, I'll open the new one. Thanks!

**Review by lmiguens-solvos — APPROVED** [review]
LGTM 👍

**Comment by dalonsod** [comment]
@pedrobaeza could you check this and merge it?

Thanks!

**Comment by lmiguens-solvos** [comment]
@astirpe I'll migrate this module to v12 during the next weeks

**Pull request merged**